docs/docs-env.yml

```diff
@@ -0,0 +1,18 @@
+name: mBuild-docs
+channels:
+  - conda-forge
+  - omnia
+  - mosdef
+dependencies:
+  - python=3.7
+  - openmm
+  - parmed
+  - pip:
```

Comment:
I think most of these packages can be installed via anaconda rather than pip. Unless there are reasons for using pip in this case that I'm not aware of, I think these should be installed from anaconda.

Reply:
Thanks.  some sphinx packages are only on PIP but I changed mostly as requested below:

name: mBuild-docs
channels:
  - conda-forge
dependencies:
  - python=3.7
  - sphinx
  - nbsphinx
  - lark-parser
  - requests
  - mock
  - pip:
    - sphinx_rtd_theme
    - sphinxcontrib-svg2pdfconverter